jenkins/templates/DeployBackend.jenkinsfile

```diff
@@ -92,6 +92,44 @@ pipeline {
 
   	}
 
+	stage('Add Production Tag'){
+
+ 		when {
+
+	        expression {
+
+                ENV == 'prod'
+
+            }
+
+        }
+
+		steps {
+
+ 			script {
+
+			    sh label: 'Docker-Tag', script: '''#!/bin/bash
+
+				# Tag image as production if deploying to prod tier
+				echo "Tagging Image as Production: $ECR_REPO:$IMAGE_TAG"
+
+				# login and get manifest
+				docker login -u AWS -p $(aws ecr get-login-password --region $REGION) $REPO_URL
+				
+				docker pull $REPO_URL:$IMAGE_TAG
+				docker tag $REPO_URL:$IMAGE_TAG $REPO_URL:prod-$IMAGE_TAG
+				docker push $REPO_URL:prod-$IMAGE_TAG
+				'''
+                
+                env.IMAGE_TAG = "prod-$IMAGE_TAG"
```

Comment:
Instead of using "env.IMAGE_TAG" set this variable as "IMAGE_TAG", this way it can be referenced in later blocks as "$IMAGE_TAG" and it will get the value you want